hack/rebase-to-origin.sh

```diff
@@ -22,24 +22,26 @@ origin_dir=$GOPATH/src/github.com/openshift/origin
 origin_s2i_godep_dir=${origin_dir}/Godeps/_workspace/src/github.com/openshift/source-to-image
 s2i_ref=$(cd ${s2i_dir} && git rev-parse --verify HEAD)
 s2i_short_ref=$(cd ${s2i_dir} && git rev-parse --short HEAD)
-s2i_godeps_ref=$(grep -m2 -A2 "openshift/source-to-image" ${origin_dir}/Godeps/Godeps.json | \
+s2i_godeps_ref=$(grep -m1 -A2 'openshift/source-to-image' ${origin_dir}/Godeps/Godeps.json | \
   grep Rev | cut -d ':' -f2 | sed -e 's/"//g' | sed -e 's/^[[:space:]]*//')
 
+source "${s2i_dir}/hack/util.sh"
```

Comment:
source this at the top, it's a good pattern to make it clearer what's being pulled in.